### ISSUE TITLE
Some buttons/axes are difficult to configure

### ISSUE THREAD

**Issue by ulti** [ihatecsv/ControlStadia#3]
When configuring L3/R3 I keep accidentally mapping to Axis 0 or Axis 2, probably because I tilt the joystick a small amount while pressing it down.

When configuring LX/LY/RX/RY, I can map LX -> Axis 0 and RX -> Axis 2 easily but it is very difficult to map LY -> Axis 1 or RY -> Axis 3.

I think you could fix this with a deadzone that ignores axis inputs close to 0.

**Comment by ihatecsv** [comment]
Yeah I was thinking about implementing something like this! I was thinking, just pick the axis that has the largest delta magnitude during the check interval. I'll take a look at it when I have a minute!
